pkg/image/registry/imagestreamtag/rest.go

```diff
@@ -333,11 +333,14 @@ func newISTag(tag string, imageStream *imageapi.ImageStream, image *imageapi.Ima
 			Name:              istagName,
 			CreationTimestamp: event.Created,
 			Annotations:       map[string]string{},
+			Labels:            imageStream.Labels,
```

Comment:
I'm hesitant copying all labels from IS. I'm not quite convinced it is the right approach...